Makefile

```diff
@@ -93,3 +92,23 @@ image/build:
 .PHONY: image/push
 image/push: image/build
 	docker push quay.io/${QUAY_ORG}/${QUAY_IMAGE}:${DEV_TAG}
+
+.PHONY: monitoring/install
+monitoring/install:
+@echo Installing service monitor in ${NAMESPACE} :
+- oc project ${NAMESPACE}
```

Comment:
```suggestion
```

`oc` hasn't been needed in this Makefile up to now, and with the below suggestions to the kubectl commands, it won't be needed in this case either.